src/Resources/Resources/ActiveDirectory/Cmdlets/NewAzureADApplicationCommand.cs

```diff
@@ -39,6 +40,7 @@ public class NewAzureADApplicationCommand : ActiveDirectoryBaseCmdlet
         [ValidateNotNullOrEmpty]
         public string DisplayName { get; set; }
 
+        [CmdletParameterBreakingChange("IdentifierUris", ChangeDescription = "The value will be considered valid only if it exists as a verified domain in a tenant.")]
```

Comment:
@dingmeng-xue How about renaming IdentifierUris to IdentifierUri at the same time? So that NewAzureADApplication could be consistent with other cmdlets.

Reply:
Good point. Since those are legacy cmdlets, we just keep them there.